packages/sdk/src/encryption/PublisherKeyExchange.ts

```diff
@@ -42,7 +42,7 @@ export class PublisherKeyExchange {
     private readonly store: LocalGroupKeyStore
     private readonly authentication: Authentication
     private readonly logger: Logger
-    private readonly erc1271ContractAddresses = new Set<EthereumAddress>()
+    private readonly erc1271ContractAddresses = new ObjectSet<Uint8Array>((u) => binaryToHex(u, true))
```

Comment:
Move back to `Set<EthereumAddress>`. Follow-up question: do we need `ObjectSet` at all?